srcpkgs/xorg-server/template

```diff
@@ -25,7 +24,7 @@ maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="MIT, BSD-3-Clause"
 homepage="https://xorg.freedesktop.org"
 distfiles="${XORG_SITE}/xserver/${pkgname}-${version}.tar.xz"
-checksum=5cc5b70b9be89443e2594b93656c60bd5e82cd7f01deb4ce4faf81dcf546a16b
+checksum=61d6aad5b6b47a116b960bd7f0cba4ee7e6da95d6bb0b127bde75d7d1acdebe5
 lib32disabled=yes
 provides="xserver-abi-extension-10_1 xserver-abi-input-24_1
```

Comment:
I believe `input` and `video` need to be bumped to 25_1.  Not sure about `extension`.  Also not entirely sure about `input`.

Reply:
Done. I assumed only the first digit is relevant, i.e. 25_1 is correct, and not 25_2, please correct me if wrong.
I kept some input* packages, as I previously fixed some lint for them, hope this is fine.

Reply:
Yes, the 25 is the import part.  The latter is just a revision and only because xbps expects/requires one.

lint fixes and updates are fine to keep.